mw-webapp/src/component/inProgress/InProgress.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from "src/component/inProgress/InProgress.module.scss";
+
+const InProgress = () => {
```

Comment:
add docs for all public members

Reply:
Sorry but don't see docs. Why you resolved this conversation?

Reply:
sorry, my bad, missed during conflicts resolving.